### ISSUE TITLE
transfer chat-drive to wechaty-plugin ?

### ISSUE THREAD

**Issue by wj-Mcat** [wechaty/chat-drive#16]
When I walk through the chat-drive source code,  I find that `file` is saved in `Message Event Listener`, which can be more easier. All of this can be a plugin for wechaty. @huan how do you think about it ?

**Comment by huan** [comment]
Yes, this should be a plugin of Wechaty, and also a Vorpal command too!

For example:

```ts
import {
  ChatDrive
}                                  from 'chat-drive'

wechaty.use(ChatDrive({
 ...
}))
```

Please feel free to send PR to this repo to improve it if you have any good ideas!
